src/services/userService.js

```diff
@@ -20,7 +22,7 @@ export function getAllUsers() {
 export function getUser(id) {
   return new User({ id })
     .fetch()
-    .then((user) => user)
+    .then((user) => user.serialize())
```

Comment:
what difference does it make?

Reply:
It removes the password from the response.